tutorials/tts/FastPitch_ChineseTTS_Training.ipynb

```diff
@@ -185,9 +185,31 @@
    "source": [
     "%%bash\n",
     "# https://ngc.nvidia.com/setup/installers/cli\n",
+    "rm -rf ngc-cli\n",
     "wget --content-disposition \"https://ngc.nvidia.com/downloads/ngccli_linux.zip\" && unzip ngccli_linux.zip && chmod u+x ngc-cli/ngc\n",
     "find ngc-cli/ -type f -exec md5sum {} + | LC_ALL=C sort | md5sum -c ngc-cli.md5\n",
-    "echo \"export PATH=\\\"\\$PATH:$(pwd)/ngc-cli\\\"\" >> ~/.bash_profile && source ~/.bash_profile"
+    "rm *.zip *.md5"
```

Comment:
For safety, should we specify the exact file names when deleting? Looks like only "ngccli_linux.zip" and "ngc-cli.md5".

Reply:
This sounds like a good idea to me.

Reply:
agreed. will do.